docs/resources.md

```diff
@@ -0,0 +1,86 @@
+# Resources Manifest
+
+The resources manifest is means of descibing which resources Ruter is sharing with operators. 
+Initially, the list will contain an audio file intended to be played when a stop is requested
+on board the vehicle. It allows us to change the sound over time and ensure a uniform aural 
+experience for passengers.
+
+This manifest should be used in a similar way as with the packages and mqtt manifests. The 
+manifest should be checked at least once a day, after 16:00, and if there is a new version
+of a resources, download and start using it. There are no contractual requirements about 
```

Comment:
Not sure if we should mention contractual requirements here. These are prone to changes and not all operators have any contractual obligations.

Reply:
Removed references to contracts and SLA, as these are subject to changes.